### PR TITLE
Adding general help rule to the README

### PR DESCRIPTION
**Pull request by nstanard**
How do you feel about this? I think it's a great idea. XD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,6 +347,17 @@ li {
...
 </table>
 
+## Getting Help
+
+Please, do not open issues for general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [StackOverflow](http://stackoverflow.com/questions/tagged/ng-bootstrap) where maintainers are looking at questions tagged with `loryJS`.
+
+StackOverflow is a much better place to ask questions since:
+* There are hundreds of people willing to help on StackOverflow
+* Questions and answers stay available for public viewing so your question / answer might help someone else
+* SO voting system assures that the best answers are prominently visible.
```
Comment (meandmax) [inline README.md]: This line is not needed in my opinion.

**Comment by meandmax** [comment]
Really good idea! Thank you @nstanard

File: README.md (hunk 1 above)
Comment (meandmax) [inline README.md]: Or keep it and change `SO voting system assures that the best answers are prominently visible.` to `The Stack Overflow voting system assures that the best answers are prominently visible.`

I needed a minute to understand what you meant with `SO` thats the reason for the change.

**Comment by nstanard** [comment]
Oops sorry. I removed it all 😬

**Comment by nstanard** [comment]
How's that @meandmax ?

**Pull request merged**